.github/workflows/github-actions-update-rules.yml

```diff
@@ -0,0 +1,49 @@
+name: Create draft PR for updated rules
+on:
+  push:
+  repository_dispatch:
+    types:
+      - set-new-golden
```

Comment:
Can't we have multiple types in one GHA and check for which type triggered the dispatch to see if we need to use `--overwrite` or not? Or some other kind of mechanism that avoids having two nearly identical GHA files?